base/precompile.jl

```diff
@@ -478,6 +478,6 @@ precompile(Base.launch, (Base.LocalManager, Dict, Array{Base.WorkerConfig, 1}, B
 precompile(Base.set_valid_processes, (Array{Int, 1}, ))
 
 # Speed up repl help
-sprint(Markdown.term, @doc mean)
+# sprint(Markdown.term, @doc mean)
```

Comment:
Oh, good catch. I commented it out because bootstrap was failing and then forgot to try putting it back in.